modules/101-cert-manager/crds/crd-certificates.yaml

```diff
@@ -100,6 +100,9 @@ spec:
                   type: array
                   items:
                     type: string
+                certificateOwnerRef:
```

Comment:
Why patching the spec is better than using an annotation?

Reply:
Because cert-manager don't use any annotations on certificate resources. Adding option to spec seems like more convenient way